avro-converter/src/main/java/io/confluent/connect/avro/AvroData.java

```diff
@@ -1043,12 +1087,15 @@ public SchemaAndValue toConnectData(org.apache.avro.Schema avroSchema, Object va
     if (value == null) {
       return null;
     }
-
-    Schema schema = (avroSchema.equals(ANYTHING_SCHEMA)) ? null : toConnectSchema(avroSchema);
-    return new SchemaAndValue(schema, toConnectData(schema, value));
+    Map<org.apache.avro.Schema, CyclicSchemaWrapper> cycleReferences = new HashMap<>();
+    Schema schema = (avroSchema.equals(ANYTHING_SCHEMA)) ? null
```

Comment:
in a multi-line ternary operator, I think is better to have each assigned value in its own line

Reply:
Also probably makes sense to declare variables in the order that they are used